python/pylibcudf/pylibcudf/libcudf/utilities/span.pxd

```diff
@@ -7,3 +7,6 @@ cdef extern from "cudf/utilities/span.hpp" namespace "cudf" nogil:
     cdef cppclass host_span[T]:
         host_span() except +
         host_span(vector[T]) except +
+    cdef cppclass device_span[T]:
```

Comment:
I think we prefer to use a `column_view` over a `device_span` (eg. we overload the `seeds` arguments in the min hashing functions to use a `column_view`. Should I delete `device_span`?